content/realtime/channels.textile

```diff
@@ -677,6 +734,12 @@ bc[swift]. let listener = channel.on { stateChange in
     print("channel state is \(stateChange.current)")
 }
 
+bc[flutter]. final stateChangeListener = channel.on()
```

Comment:
As above

Reply:
addressed in e288898cb0f972b74350bd69c2da054346fbb4f0